src/e2e/java/uk/gov/hmcts/reform/sscs/functional/ccd/CcdMidEventControllerFunctionalTest.java

```diff
@@ -53,46 +55,53 @@ public void shouldReturn500ForUnauthorised() throws IOException {
         HttpResponse httpResponse = sscsMyaBackendRequests.midEvent(new StringEntity(getJsonCallbackForTest("handlers/writefinaldecision/writeFinalDecisionCallback.json")), "PreviewFinalDecision");
         CcdEventResponse ccdEventResponse = getCcdEventResponse(httpResponse);
-        assertThat(httpResponse.getStatusLine().getStatusCode(), is(200));
-        assertThat(ccdEventResponse.getData().getSscsFinalDecisionCaseData().getWriteFinalDecisionPreviewDocument(), is(not(nullValue())));
+        assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(200);
+        assertThat(ccdEventResponse.getData().getSscsFinalDecisionCaseData().getWriteFinalDecisionPreviewDocument()).isNotNull();
     }
 
+    @DisplayName("Preview adjourn case should populate adjourn case preview document")
     @Test
-    public void previewAdjournCase_shouldPopulateAdjournCasePreviewDocument() throws IOException {
-        HttpResponse httpResponse = sscsMyaBackendRequests.midEvent(new StringEntity(getJsonCallbackForTest("handlers/adjourncase/adjournCaseCallback.json")), "PreviewAdjournCase");
+    public void testPreviewAdjournCaseGaps() throws IOException {
+        HttpResponse httpResponse = sscsMyaBackendRequests.midEvent(new StringEntity(getJsonCallbackForTest(
+            "handlers/adjourncase/adjournCaseGapsCallback.json")), "PreviewAdjournCase");
         CcdEventResponse ccdEventResponse = getCcdEventResponse(httpResponse);
-        assertThat(httpResponse.getStatusLine().getStatusCode(), is(200));
-        assertThat(ccdEventResponse.getData().getAdjournCasePreviewDocument(), is(not(nullValue())));
+        assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(200);
+        assertThat(ccdEventResponse.getData().getAdjournCasePreviewDocument()).isNotNull();
     }
 
+    @DisplayName("Admin restore cases should return an error for data without restore cases date")
     @Test
-    public void adminRestoreCases_shouldReturnAnErrorForDataWithoutRestoreCasesDate() throws IOException {
-        HttpResponse httpResponse = sscsMyaBackendRequests.midEvent(new StringEntity(getJsonCallbackForTest("handlers/adjourncase/adjournCaseCallback.json")), "AdminRestoreCases");
+    public void testAdminRestoreCasesGaps() throws IOException {
+        HttpResponse httpResponse = sscsMyaBackendRequests.midEvent(new StringEntity(getJsonCallbackForTest(
+            "handlers/adjourncase/adjournCaseGapsCallback.json")), "AdminRestoreCases");
         CcdEventResponse ccdEventResponse = getCcdEventResponse(httpResponse);
-        assertThat(httpResponse.getStatusLine().getStatusCode(), is(200));
-        assertThat(ccdEventResponse.getWarnings().size(), is(0));
-        assertThat(ccdEventResponse.getErrors().size(), is(1));
-        assertThat(ccdEventResponse.getErrors().iterator().next(), is("Unable to extract restoreCaseFileName"));
+        assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(200);
```

Comment:
I would extract out these magic numbers, instead of 200 do something like 
```suggestion
        assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(HttpStatus.SC_OK);
```

Reply:
done